### DISCUSSION TITLE
台積電赴美 王美花：3奈米以下留台

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#856] [General]
我覺得台積電移往美國算是意料之中吧，畢竟台灣政局不穩，沒有商人會希望把最重要的製程都留在這樣的地方，雖然新聞說最先進到會留在台灣，但我還是蠻懷疑的
網址：https://tw.news.yahoo.com/台積電赴美-王美花-3奈米以下留台-201000968.html

**Comment by Pietra1226** [comment]
感覺台積電的輪班生態在美國不太受歡迎，但五奈米技術的轉移對台灣矽盾強度減弱了一些，遲早都會轉移的只是時間問題。

**Comment by Chiway1229** [comment]
這種製程上的突破是常有的事，說不定3奈米的製程在五年被就能被突破
台灣可能不太能只靠一座護國神山在撐經濟
要能發展出多元優勢才行

**Reply by chia-kai-chang** [reply]
沒錯，ICT產業越來越熱門，所以才一直要大家學Python阿

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
不知道不久的將來 台灣的半導體產業
能否像現在這樣彭勃發展

**Comment by williamyen0623** [comment]
台灣需要發展的不只有半導體產業
很多新興科技產業的發展也需要政府的重視
才能多元化產業項目
不能總是依靠台積電

**Comment by CPCpyt** [comment]
轉移本來就是未來看好的前景，
地緣政治穩定、國家強勢，真的很難有不遷往的理由！
